syft/frameworks/crypten/__init__.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import syft
+
+from syft.frameworks.crypten.context import run_party
+
+import crypten.communicator as comm
+import crypten
+
+
+def load(tag: str, src: int):
+    if src == comm.get().get_rank():
+        # Means the data is on one of our local workers
+
+        worker = syft.local_worker.get_worker_from_rank(src)
```

Comment:
@youben11 I changed the load to look after the worker given a rank (the PR I merged into the CrypTen branch for plans did not have those changes and it fails when it looks for the tagged data on the ```syft.local_worker```)